WcaOnRails/app/views/shared/_associated_events_picker.html.erb

```diff
@@ -23,7 +23,7 @@ Note: form_builder.object is the object having associated events.
 
 <div class="form-group <%= "has-error" unless errors.empty? %>">
   <%= label_tag events_association_name, class: "associated-events-label" do %>
-    <%= label %>
+    <%= label %> (<span class="events-selected-count"></span> selected)
```

Comment:
Yes, I actually was working on translating this while you were reviewing, sorry about that! The current wording doesn't change based on count at least in English, do you know if that would be a problem in other languages?